pipeline/load.py

```diff
@@ -0,0 +1,331 @@
+        scientific_id = cur.fetchone()[0]
+
+        return scientific_id
+
+
+def insert_botanists(conn: 'connection', plant_df: pd.DataFrame) -> None:
+    """Inserts Botanist data into the relevant tables in the database
+        Ignores duplicate entries"""
+    with conn.cursor() as cur:
+        for index, row in plant_df.iterrows():
+            first_name = row["First Name"]
+            last_name = row["Last Name"]
+            email = row["botanist.email"]
+            phone = row["botanist.phone"]
+
+            cur.execute("""
+                SELECT COUNT(*)
+                FROM delta.Botanists
+                WHERE first_name = %s AND last_name = %s AND email = %s AND phone = %s;
+            """, (first_name, last_name, email, phone))
+
+            count = cur.fetchone()[0]
+
+            if count == 0:
+                cur.execute("""
+                    INSERT INTO delta.Botanists (first_name, last_name, email, phone)
+                    VALUES (%s, %s, %s, %s);
+                """, (first_name, last_name, email, phone))
+                print(f"Inserted: {first_name} {last_name}, {email}, {phone}")
+            else:
+                print(f"""Duplicate found: {first_name} {
+                      last_name}, {email}, {phone}""")
+
+
+def insert_scientific_name(conn: 'connection', plant_df: pd.DataFrame) -> None:
+    """Inserts the scientific name of a plant into the database
+        Ignores duplicate entries"""
+    with conn.cursor() as cur:
+        for index, row in plant_df.iterrows():
+            scientific_name = row["scientific_name"]
+            if pd.isna(scientific_name):
+                scientific_name = "None"
+            cur.execute("""
+                SELECT COUNT(*)
+                FROM delta.Scientific_Names
+                WHERE scientific_name = %s;
+            """, scientific_name)
+
+            count = cur.fetchone()[0]
+
+            if count == 0:
+                cur.execute("""
+                    INSERT INTO delta.Scientific_Names (scientific_name)
+                    VALUES (%s);
+                """, (scientific_name))
+                print(f"Inserted: {scientific_name}")
+            else:
+                print(f"Duplicate: {scientific_name}")
+
+
+def insert_location(conn: 'connection', plant_df: pd.DataFrame) -> None:
+    """Inserts the location of a plant into the database
+        Ignores duplicate entries"""
+    with conn.cursor() as cur:
+        for index, row in plant_df.iterrows():
+            latitude = row["Latitude"]
+            longitude = row["Longitude"]
+            town = row["Town"]
+            country_code = row["Country_Code"]
+            continent = row["Continent"]
+            city = row["City"]
+
+            cur.execute("""SELECT continent_id
+                        FROM delta.Continents
+                        WHERE continent_name = %s;
+                        """, (continent))
+            continent_id = cur.fetchone()[0]
```

Comment:
we only have 7 continent ids. Instead of calling this query multiple times in this for loop, could we run it once before and save the key/values elsewhere.